src/components/PuppyToken.jsx

```diff
@@ -0,0 +1,250 @@
+import React, { useContext, useState } from 'react'
+import { useToggle } from 'react-use'
+import { transactionContext } from '../context/TransactionContext'
+import Swal from 'sweetalert'
+
+const ProcessingButton = () => (
+  <button
+    type="button"
+    disabled
+    className="flex items-center justify-center bg-[#3d4f7c] text-white w-full mt-2 border-[1px] p-2 border-[#8e9fc9d5] rounded-full cursor-not-allowed"
+  >
+    <svg
+      className="animate-spin w-5 mr-2 text-white"
+      xmlns="http://www.w3.org/2000/svg"
+      fill="none"
+      viewBox="0 0 24 24"
+    >
+      <circle
+        className="opacity-25"
+        cx="12"
+        cy="12"
+        r="10"
+        stroke="currentColor"
+        stroke-width="4"
```

Comment:
```suggestion
        strokeWidth="4"
```

We should change it to camelCase when using Jsx.

Reply:
Addressed in [98d2d4b](https://github.com/lohsuan/puppy_sponsor/pull/14/commits/98d2d4bd70c5ed8f12af1e29d0f036c7ecd8f64a).